.github/CONTRIBUTING.md

```diff
@@ -14,3 +14,14 @@
 * If you would like to work on an issue, drop in a comment at the issue. If it is already assigned to someone, but there is no sign of any work being done, please free to drop in a comment so that the issue can be assigned to you if the previous assignee has dropped it entirely.
 
 Do read the [Open Source Developer Guide and Best Practices at FOSSASIA](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
+
+**Review Policy**
```

Comment:
Any idea about the person who opens the PR can merge it or not after waiting for review for a certain period of time?